src/Evolution/Systems/Cce/WorldtubeBufferUpdater.cpp

```diff
@@ -295,13 +299,14 @@ BondiWorldtubeH5BufferUpdater::BondiWorldtubeH5BufferUpdater(
   const std::string text_radius =
       cce_data_filename.substr(r_pos + 1, dot_pos - r_pos - 1);
   try {
-    extraction_radius_ = stod(text_radius);
+    extraction_radius_ = static_cast<bool>(extraction_radius)
+                             ? *extraction_radius
+                             : stod(text_radius);
```

Comment:
`std::stod`?

Reply:
you seem to have replaced one stod (on line 114) with std::stod but not the other :P

Reply:
d'oh. fixed.